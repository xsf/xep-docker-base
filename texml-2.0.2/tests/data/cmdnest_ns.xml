<TeXML xmlns="http://getfo.sourceforge.net/texml/ns1">
	<cmd name="par1">
		<opt>
			<cmd name="par2">
				<opt>
					<cmd name="par3">
						<parm>
							<cmd name="par4"/>
						</parm>
						<opt>&#1984;</opt>
					</cmd>
				</opt>
			</cmd>
		</opt>
		<parm>aaa</parm>
	</cmd>
</TeXML>
