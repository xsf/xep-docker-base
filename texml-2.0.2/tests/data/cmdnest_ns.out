\par1[ \par2[ \par3{ \par4{} }[\unicodechar{1984}] ] ]{aaa}
